### PR TITLE
Proper v12 typings

### PR DESCRIPTION
**Pull request by iCrawl**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -388,46 +460,75 @@ declare module 'discord.js' {
...
+		public setAFKChannel(afkChannel: ChannelResolvable, reason?: string): Promise<Guild>;
+		public setAFKTimeout(afkTimeout: number, reason?: string): Promise<Guild>;
 		public setChannelPosition(channel: string | GuildChannel, position: number, relative?: boolean): Promise<Guild>;
 		public setChannelPositions(channelPositions: ChannelPosition[]): Promise<Guild>;
-		public setExcplicitContentFilter(explicitContentFilter: number): Promise<Guild>;
-		public setIcon(icon: Base64Resolvable): Promise<Guild>;
-		public setName(name: string): Promise<Guild>;
-		public setOwner(owner: GuildMemberResolvable): Promise<Guild>;
-		public setRegion(region: string): Promise<Guild>;
+		public setExcplicitContentFilter(explicitContentFilter: number, reason?: string): Promise<Guild>;
```
Comment (PgBiel) [inline index.d.ts]: "Excplicit" is not correct

**Review by PgBiel — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (iCrawl) [reply index.d.ts]: tfw

**Comment by PgBiel** [comment]
should probably look the data store thing

**Comment by PgBiel** [comment]
🤔

**Comment by kzay** [comment]
There is some issue, like the users type in Client objects (now is UserStore)

**Comment by zajrik** [comment]
Going to go ahead and merge this for now. Looks like we still need DataStore stuff but that's going to be generics I believe so I'll have to tackle it

**Pull request merged**